pkg/config/templates/config.toml

```diff
@@ -98,6 +98,13 @@ additional_redirect_urls = ["https://127.0.0.1:3000"]
 # How long tokens are valid for, in seconds. Defaults to 3600 (1 hour), maximum 604,800 (1 week).
 jwt_expiry = 3600
 # If disabled, the refresh token will never expire.
+minimum_password_length = 6
+# Passwords shorter than this value will be rejected as weak. Minimum 6, recommended 8 or more.
+password_requirements = ""
+# Passwords that do not have at least one of each will be rejected as weak.
+# Letters and digits: abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ:0123456789
+# Lowercase, uppercase letters and digits: abcdefghijklmnopqrstuvwxyz:ABCDEFGHIJKLMNOPQRSTUVWXYZ:0123456789
+# Lowercase, uppercase letters, digits and symbols: abcdefghijklmnopqrstuvwxyz:ABCDEFGHIJKLMNOPQRSTUVWXYZ:0123456789:!@#$%^&*()_+-=[]{};'\\\\:\"|<>?,./`~
```

Comment:
We should make this an enum and validate it, similar to frontend.

<img width="584" alt="Screenshot 2024-11-18 at 3 11 03 PM" src="https://github.com/user-attachments/assets/54d309a2-a8df-44a6-acef-606fe4d02659">

Reply:
Sounds good to me